src/coreclr/debug/daccess/gcdump_dac.cpp

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include <stdafx.h>
+
+/* There is no DAC build of gcdump, so instead
```

Comment:
You mean you didn't think `REVISIT_TODO Mon 10/10/2005` from nidump.cpp was useful to keep?

Reply:
I figured if it wasn't fixed by now, it's not going to be fixed haha